psi4/src/psi4/libscf_solver/rohf.cc

```diff
@@ -512,36 +481,28 @@ void ROHF::form_D()
         double** Da = Da_->pointer(h);
         double** Db = Db_->pointer(h);
 
-        if (na == 0)
-            memset(static_cast<void*>(Da[0]), '\0', sizeof(double)*nso*nso);
-        if (nb == 0)
-            memset(static_cast<void*>(Db[0]), '\0', sizeof(double)*nso*nso);
-
-
-        C_DGEMM('N','T',nso,nso,na,1.0,Ca[0],nmo,Ca[0],nmo,0.0,Da[0],nso);
-        C_DGEMM('N','T',nso,nso,nb,1.0,Ca[0],nmo,Ca[0],nmo,0.0,Db[0],nso);
+        if (na == 0) memset(static_cast<void*>(Da[0]), '\0', sizeof(double) * nso * nso);
+        if (nb == 0) memset(static_cast<void*>(Db[0]), '\0', sizeof(double) * nso * nso);
```

Comment:
Nitpick, but while I see it. Can we do `Da_->zero(); Db_->zero();` above the `nirrep_` loop and remove the memset?